server/utils/nunjucksSetup.ts

```diff
@@ -1,38 +1,38 @@
 /* eslint-disable no-param-reassign */
 /* istanbul ignore file */
 
-import * as pathModule from 'path'
-import nunjucks from 'nunjucks'
 import express from 'express'
+import nunjucks from 'nunjucks'
+import * as pathModule from 'path'
 
-import { PersonStatus, ErrorMessages } from '@approved-premises/ui'
+import { ErrorMessages, PersonStatus } from '@approved-premises/ui'
+import config from '../config'
 import applicationPaths from '../paths/apply'
 import assessPaths from '../paths/assess'
 import reportPaths from '../paths/report'
-import config from '../config'
-import { initialiseName, removeBlankSummaryListItems } from './utils'
 import staticPaths from '../paths/static'
 import {
   documentSummaryListRows,
   inProgressApplicationTableRows,
   submittedApplicationTableRows,
 } from './applicationUtils'
-import * as TaskListUtils from './taskListUtils'
-import * as OasysImportUtils from './oasysImportUtils'
-import { dateFieldValues } from './formUtils'
-import { checkYourAnswersSections, getApplicantDetails } from './checkYourAnswersUtils'
-import { DateFormats } from './dateUtils'
 import { getApplicationTimelineEvents } from './applications/utils'
 import { applicationStatusRadios } from './assessUtils'
+import { checkYourAnswersSections, getApplicantDetails } from './checkYourAnswersUtils'
+import { DateFormats } from './dateUtils'
+import { dateFieldValues } from './formUtils'
+import * as OasysImportUtils from './oasysImportUtils'
 import { statusTag } from './personUtils'
+import * as TaskListUtils from './taskListUtils'
+import { initialiseName, removeBlankSummaryListItems } from './utils'
 
 const production = process.env.NODE_ENV === 'production'
 
 export default function nunjucksSetup(app: express.Express, path: pathModule.PlatformPath): void {
   app.set('view engine', 'njk')
 
   app.locals.asset_path = '/assets/'
-  app.locals.applicationName = 'CAS-2'
+  app.locals.applicationName = 'Short-Term Accommodation (CAS 2)'
```

Comment:
With `short-term-accommodation-cas-2` as the domain name we need to choose how we present the service name that appears in the title. In particular what do we do about the "CAS-2" part? 

1. Short-Term Accommodation (CAS 2)
2. Short-Term Accommodation - CAS 2
3. Short-Term Accommodation CAS 2
4. Other?

"CAS-2" appears throughout as a secondary/short-hand name so it seemed reasonable to put it in brackets here as a way to connect the long and short hand name?

Reply:
@chaneldiepmoj what do you think?

Reply:
[Conversation continued in the Slack thread](https://mojdt.slack.com/archives/C04T6SC31HA/p1704821481047049?thread_ts=1704812197.680799&cid=C04T6SC31HA)